### PR TITLE
Add Run configuration

### PR DESCRIPTION
**Pull request by Felanbird**
~~When setting up Run configurations on Intellij, said configuration can be saved (or imported) to the `.run` folder.~~

~~This PR adds that folder to `.gitignore` so the configuration script isn't always leering over you when you make changes.~~

this PR adds the config instead since it's nicer to just be able to run the plugin, instead of having to set it up yourself.

### CODE REVIEW

**Comment by Mm2PL** [comment]
> When setting up Run configurations on Intellij, said configuration can be saved (or imported) to the .run folder.

Isn't that only for configurations marked to be shared through version control. I don't even have a `.run` folder but I do have a run configuration.

**Comment by Felanbird** [comment]
> Isn't that only for configurations marked to be shared through version control. I don't even have a `.run` folder but I do have a run configuration.

It comes from saving said config as a project file, but if you import an existing one from another plugin, that option will be on by default. (since you are importing a project file)

I could unsave it but I have no idea where Intellij saves the info normally 🤔 

image

**Comment by Mm2PL** [comment]
image
If you uncheck that option it will save the config somewhere in .idea directory (which is already gitignored)

File: .run/Run Dink.run.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    <option name="ALTERNATIVE_JRE_PATH_ENABLED" value="true" />
+    <option name="MAIN_CLASS_NAME" value="dinkplugin.DinkTest" />
+    <module name="dinkplugin.test" />
+    <option name="PROGRAM_PARAMETERS" value="--debug --developer-mode --illegal-access=warn" />
+    <option name="VM_PARAMETERS" value="-ea" />
+    <method v="2">
+      <option name="Make" enabled="true" />
+    </method>
+  </configuration>
+</component>
```
Comment (Felanbird) [inline .run/Run Dink.run.xml:13]: when u toggle the setting it generates without a newline

**Comment by pajlada** [comment]
people could add to their local gitignore if they need to if they have a special setup, or just copy this configuration and store it in a non-local directory

**Pull request merged**